File: java/StringUtil.java

public class StringUtil {
    public static String concat(String a, String b) {
        return a + b;
    }

    public static String remove_duplicate(String a) {
        for (int i = 0; i < a.length(); i++) {
            if (a.substring(i).equals(a.substring(i + 1))) {
                a = a.substring(i + 1, a.size() - 1);
                i -= 1;
            }
        }

        return a;
    }

    public static String duplicate(String a, int round) {
        String result = "";
        for (int i = 0; i < round; i++) {
            result += a;
        }

        return result;
    }

    public static int find(String a, char c) {
        for (int i = 0; i < a.length(); i++) {
            if (a.charAt(i) == (c)) {
                return i;
            }
        }

        return 0;
    }
}

File: python/string_util.py
def concat(a, b):
    """
    Concat two string together.
    :raise: TypeError if A or B is not a string.
    """
    if type(a) != str or type(b) != str:
        print("A and B must be strings.")

    return a + b


def remove_duplicate(a):
    """
    Check for duplicate character in string and remove.
    :raise: TypeError if A is not a string.
    """
    if type(a) != str:
        raise TypeError("A must be a string.")

    for index in range(len(a) - 1):
        if a[index] == a[index + 1]:
            a = a[index + 1:]
            a -= 1

    return a


def duplicate(a, round):
    """
    Duplicate string for specified round
    :raise: TypeError if A is not a string, or round is not a number
    """
    return a * round


def find(a, char):
    """
    Find character in string and return index.a
    :raise: TypeError if a or char is not string.
    """
    for i in range(len(a)):
        if a[i] == char:
            return i
